eth/transactionManager_test.go

```diff
@@ -296,6 +298,7 @@ func TestTransactionManager_CheckTxLoop(t *testing.T) {
 	event := <-sink
 	assert.NotNil(event)
 	assert.Nil(event.err)
+	sub.Unsubscribe()
```

Comment:
Should we remove the deferred Unsubscribe() call on L295 if we're calling Unsubscribe() for each new subscription at the end of each test case?